src/explorer/webview.ts

```diff
@@ -149,46 +145,15 @@ export class ExplorerWebview {
      * Cleans up the result details section, removing event handlers and elements
      */
     private cleanUpResultDetails(): void {
-        // Remove event handlers
         const tabContainer: HTMLElement | undefined = getOptionalDocumentElementById(document, "tabcontainer", HTMLElement);
-        if (tabContainer) {
-            for (const tabContainerElement of getElementChildren(tabContainer, HTMLDivElement)) {
-                tabContainerElement.removeEventListener("click", this.onTabClicked.bind(this));
-            }
-        }
-
-        const sourceLinks: HTMLCollectionOf<HTMLAnchorElement> = getDocumentElementsByClassName(document, "sourcelink", HTMLAnchorElement);
-        for (const sourceLink of sourceLinks) {
-            sourceLink.removeEventListener("click", this.onSourceLinkClicked);
-        }
-
-        if (this.hasCodeFlows) {
-            const codeFlowTrees: HTMLCollectionOf<HTMLUListElement> = getDocumentElementsByClassName(document, "codeflowtreeroot", HTMLUListElement);
-            for (const codeFlowTree of codeFlowTrees) {
-                codeFlowTree.removeEventListener("click", this.onCodeFlowTreeClicked.bind(this));
-            }
-
-            let element: HTMLDivElement = getDocumentElementById(document, "expandallcodeflow", HTMLDivElement);
-            element.removeEventListener("click", this.onCollapseAllClicked.bind(this));
-
-            element = getDocumentElementById(document, "collapseallcodeflow", HTMLDivElement);
-            element.removeEventListener("click", this.onCollapseAllClicked.bind(this));
-
-            const verbosityElement: HTMLInputElement = getDocumentElementById(document, "codeflowverbosity", HTMLInputElement);
-            verbosityElement.removeEventListener("click", this.onVerbosityChange.bind(this));
+        if (!tabContainer) {
```

Comment:
Most of it was deleted.  Why would you leave it alone?

---
In reply to: [405126272](https://github.com/microsoft/sarif-vscode-extension/pull/218#discussion_r405126272) [](ancestors = 405126272)

Reply:
@roywill: A few reasons:
1. How do know deleting this code results in equivalent behavior?
2. Gabe and I agreed to split work between WebView (me) and non-WebView (Gabe) parts. 3. Per #2, this view will be replaced soon. Why spend time on stuff we're throwing out?

Reply:
This code was modified because the web-view was not working as expected due to variables not being properly set when you moved between active diagnostics, etc. It was impeding my manual testing.

---
In reply to: [405171563](https://github.com/microsoft/sarif-vscode-extension/pull/218#discussion_r405171563) [](ancestors = 405171563,405126272)

Reply:
This code was modified because the web-view was not working as expected due to variables not being properly set when you moved between active diagnostics, etc. It was impeding my manual testing.

---
In reply to: [405181046](https://github.com/microsoft/sarif-vscode-extension/pull/218#discussion_r405181046) [](ancestors = 405181046)

Reply:
Thanks for the explainer.